src/validations/linters.js

```diff
@@ -4,9 +4,9 @@ import htmllint from 'htmllint';
 import HTMLInspector from 'html-inspector';
 import {JSHINT as jshint} from 'jshint';
 import prettyCSS from 'PrettyCSS';
-import stylelint from 'stylelint';
 import Slowparse from 'slowparse/src';
 import SlowparseHTMLParser from 'slowparse/src/HTMLParser';
+import stylelint from '../util/minimalStylelint';
```

Comment:
This is a function that will return a promise:

* If the CSS is syntactically valid, the promise will resolve with a `result` value containing a `messages` property listing lint errors.
* If the CSS is not syntactically valid, the promise will reject with a `CssSyntaxError` that has details on the problem.